src/Nethermind/Nethermind.Serialization.Json/UInt256Converter.cs

```diff
@@ -29,8 +29,18 @@ private static UInt256 ReadInternal(ref Utf8JsonReader reader, JsonTokenType all
     {
         if (reader.TokenType == JsonTokenType.Number)
         {
-            return reader.GetUInt64();
+            if (reader.TryGetUInt64(out ulong smallValue))
+            {
+                return smallValue;
+            }
+
+            ReadOnlySpan<byte> bytes = reader.HasValueSequence ? reader.ValueSequence.ToArray() : reader.ValueSpan;
```

Comment:
`reader.ValueSequence.ToArray()` probably won't happen to often and might not be the biggest cost, but always there's a way to `stackalloc byte[expected]` and copying it there.